blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
+3. Navigate to **Toolbox** > **Bot** and drag a **Call Bot Connector** action to the editor.
 
 	![GC Toolbox](images/Toolbox.png "GC Toolbox")
-3. In the options of the Call Bot Connector block, you should now be able to select your Bot Integration, Bot Name and Bot Version as specified in the API call we made earlier
+4. Configure the action for the **Name**, **Bot Integration**, and **Bot Version** as specified in the REST API call.
 
 	![Call Bot Connector block](images/CallConnector.png "Call Bot Connector block")
-4. Create the rest of the flow as you would for other bots
+5. Complete the rest of the flow as required. For more information, see [Call Bot Connector action](https://help.mypurecloud.com/articles/call-bot-connector-action/ "Opens the Call Bot Connector action page") in the Genesys Cloud Resource Center.
 
 	![Finish Architect Flow](images/ArchitectFlow.png "Finish Architect Flow")
 
-###Set up Web Messaging
+### Set up Web Messaging and test the bot
 
-1. Create a configuration in any way you like
-2. Create a deployment
-	1.Send messages to your message flow
-	
-	![Messenger Deployment](images/MessengerDep.png "Messenger Deployment")
-	2. Copy the snippet from the Deployment and deploy to a web page to test. The index.html page in the code has a sample to use.
+1. Configure a messenger to interact with the bot. For more information, see [Configure Messenger](https://help.mypurecloud.com/articles/configure-messenger/ "Opens the Configure Messenger page") in the Genesys Cloud Resource Center.
+2. Create a messenger deployment with the Architect flow created for the bot. For more information, see [Deploy Messenger](https://help.mypurecloud.com/articles/deploy-messenger/ "Opens the Deploy Messenger page") in the Genesys Cloud Resource Center.
+   ![Messenger Deployment](images/MessengerDep.png "Messenger Deployment")
+3. To deploy the Messenger snippet to your website, copy the snippet under **Deploy your snippet** and paste the snippet to the `<head>` tag of all of your webpages. The `index.html` page in the code has a sample to use.
 
 	![Messenger](images/Messenger.png "Messenger")
 	
 
 ## Additional resources
 
-[comment]: # Do not list any links mentioned in the blueprint. List any links to Genesys or third-party material that provide further background. Format and examples follow.
-
-* [About Genesys Bot Connector](https://help.mypurecloud.com/articles/about-genesys-bot-connector/ "Opens the About Genesys Bot Connector document")
-* [About Web Messaging](https://help.mypurecloud.com/articles/about-web-messaging/ "Opens the About Web Messaging document")
\ No newline at end of file
+* [About Genesys Bot Connector](https://help.mypurecloud.com/articles/about-genesys-bot-connector/ "Opens the About Genesys Bot Connector page") in Genesys Cloud Resource Center
+* [About Web Messaging](https://help.mypurecloud.com/articles/about-web-messaging/ "Opens the About Web Messaging page") in the Genesys Cloud Resource Center
```

Comment:
Add a link to the Messenger JavaScript SDK: https://developer.genesys.cloud/api/digital/webmessaging/messengersdk/

Add a link to the web messaging Guest API: https://developer.genesys.cloud/api/digital/webmessaging/websocketapi